dist/debian/control

```diff
@@ -15,14 +15,15 @@ Build-Depends:
  libssl-dev,
  python,
  uuid-dev,
+ golang-go (>= 2:1.13)
 Standards-Version: 3.9.8
 Homepage: http://gmkurtzer.github.io/singularity
 Vcs-Git: https://github.com/hpcng/singularity.git
 Vcs-Browser: https://github.com/hpcng/singularity
 
 # "singularity" is a packaged game (but the contents don't clash)
 Package: singularity-container
-Architecture: any
+Architecture: amd64
```

Comment:
What happens if you don't make this change?  I still have an i386 debian machine and it complains about this.

Reply:
I did not expect, that a tool designed for HPC like singularity is useful on 32bit. Maybe I can find a 32bit Debian to test it.

Reply:
It's probably not very useful, it's just that it's the machine I have handy!  You could leave that testing to me, but what happens on your machine if you leave it as "any"?  debuild appears to figure out what architecture to put in there.

Reply:
By the way, the semantic of the Architecture line is not the build target.

A value of `all` means that it is an **architecture-independent** package, which can be installed on all architectures. For example a package which only contains bash-scripts. 

This is not true for singularity-container package. It is ether amd64 or i386, depending on the architecture the package is build. It would need two separate Debian packages, one for amd64 and one for i386 (which needs to be built on a i386 or the compiler has to produce i386 binaries.) But this is not the task of "debuild", this must be a configuration in the sourcecode (or mconfig) of singularity.

Reply:
The value here though is not `all`, it is `any`.  Does that cause something to break?